backend/serverless.yml

```diff
@@ -58,6 +58,22 @@ functions:
           method: get
           path: /api/v1/courses/{courseId}
 
+  #
+  # Grade endpoints
+  #
+  getSessionsGrades:
+    handler: src/handlers/grade.getSessionsGrades
+    events:
+      - http:
+          method: get
+          path: /api/v1/courses/{courseId}/grades
+  getQuestionsGrades:
```

Comment:
Shouldnt these be flipped?

Reply:
no i think it's correct, although the naming is a bit confusing. getQuestionGrades is when you drill down into a specific session